app/utils/file/index.ts

```diff
@@ -60,20 +60,36 @@ const SUPPORTED_VIDEO_FORMAT = Platform.select({
 const types: Record<string, string> = {};
 const extensions: Record<string, readonly string[]> = {};
 
-export function filterFiles<T extends FileModel | FileInfo>(files: T[], filter: FileFilter) {
+export function filterFileExtensions(filter: FileFilter): string {
+    let searchTerms: string[] = [];
     switch (filter) {
         case 'all':
-            return files;
-        case 'videos':
-            return files.filter((f) => isVideo(f));
+            return '';
         case 'documents':
-            return files.filter((f) => isDocument(f));
+            searchTerms = Files.DOCUMENT_TYPES;
+            break;
+        case 'spreadsheets':
```

Comment:
Each one of these should be a constant

Reply:
add here and modified other two locations that used these values. Thanks for the request. https://github.com/mattermost/mattermost-mobile/pull/6366/commits/a2bdc84b98db9172f67096a956230ebee96c6d29